airbase/pom.xml

```diff
@@ -440,6 +437,9 @@
                         <!-- ${argLine} is for Jacoco: https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html -->
                         <argLine>
                             ${argLine}
+                            -Duser.timezone=${air.test.timezone}
```

Comment:
add a code comment why this goes in argsLine and not systemPropertyVariables

Reply:
Added :)